### PR TITLE
Add the final routes for Newspaper integration

### PR DESCRIPTION
**Pull request by hmgibson23**
This adds the last remaining routes for newspaper integration. 

The routes are:
```
http://r2.code.dev-gutools.co.uk/tools/newspaperintegration/tags/new/:since
http://r2.code.dev-gutools.co.uk/tools/newspaperintegration/tags/modified/:since
http://r2.code.dev-gutools.co.uk/tools/newspaperintegration/tags/deleted/:since
```

### CODE REVIEW

File: app/controllers/ReadOnly.scala [hunk 1]
```diff
@@ -23,13 +24,50 @@ object ReadOnlyApi extends Controller {
...
-      {merges.map( x => x.asExportedXml)}
+    Ok(
+      <merges>
+        {merges.map( x => x.asExportedXml)}
       </merges>
     )
   }
+
+  def deletesAsXml(since: Long) = Action {
+    val deletes = JobRepository.getDeletes.map { job =>
```
Comment (clloyd) [inline app/controllers/ReadOnly.scala]: This isn't a record of all Jobs that have happened in the past, they are deleted once complete (the only reason they're there now is due to other issues with merge/delete)

Tag Audit is a better source of this information

Comment (hmgibson23) [inline app/controllers/ReadOnly.scala]: Okay cool - will update

File: app/controllers/TagManagementApi.scala [hunk 2]
```diff
@@ -161,7 +161,7 @@ object TagManagementApi extends Controller with PanDomainAuthActions {
     }
   }
 
-  def deleteTag(id: Long)= (APIAuthAction andThen DeleteTagPermissionsCheck) { req =>
```
Comment (clloyd) [inline app/controllers/TagManagementApi.scala]: Why are we removing this permission check?

File: conf/routes [hunk 3]
```diff
@@ -51,10 +51,17 @@ GET        /hyper/tags                    controllers.HyperMediaApi.tags
 GET        /hyper/tags/:id                controllers.HyperMediaApi.tag(id: Long)
 OPTIONS    /*all                          controllers.HyperMediaApi.preflight(all: String)
 
+# Tag export for inCopy
 GET        /tags/export/all               controllers.ReadOnlyApi.getTagsAsXml
 GET        /tags/export/single/:id        controllers.ReadOnlyApi.tagAsXml(id: Long)
 GET        /tags/export/merges/:since     controllers.ReadOnlyApi.mergesAsXml(since: Long)
 
+# More newspaper integration
+GET       /tools/newspaperintegration/tags/modified/:since          controllers.ReadOnlyApi.modifiedAsXml(since: Long)
```
Comment (clloyd) [inline conf/routes]: We are going to have to change these urls in the consumers, so if we want something less ugly :D

Comment (hmgibson23) [inline conf/routes]: I guess so. I imagine for now they'll just want to switch over the hosts though

**Comment by hmgibson23** [comment]
@clloyd - mind taking a look?

**Comment by clloyd** [comment]
:+1:

**Pull request merged**